tasks.py

```diff
@@ -552,20 +552,33 @@ def lint(ctx, env="container"):
 
 
 @task(help={
-    "name": "name of the kind cluster to test.",
-    "export": "where to export kind logs"
+    "name": "name of the kind cluster to test (only kind uses).",
+    "export": "where to export kind logs.",
+    "kubeconfig_path": "kubeconfig path. By default, load kubeconfig from kind.",
```

Comment:
Suggest:
` "kubeconfig": "kubeconfig location. By default, use the kubeconfig from kind.",`

This will require some naming changes below but I think it looks better

Reply:
changed.